src/utils/scratchOrgUtils.ts

```diff
@@ -281,6 +286,21 @@ export default class ScratchOrgUtils {
     hubOrg: Org
   ): Promise<boolean> {
     let hubConn = hubOrg.getConnection();
+    
+    //Check that SfdxAuthUrl__c field exists to avoid a breaking change
```

Comment:
I've added a describe on `ScratchOrgInfo` to be sure that the new `SfdxAuthUrl__c` field exists, so that it won't break users that have not yet upgraded to the new version of the `sfpower-scratchorg-pool` Unlocked Package.

Not sure if there is a better way to do this than describing the object and looking at the fields name one by one.

Reply:
Yes thats right, describe is the way to go.  We will need to add on to this function or create a similar one to check whether this field exists, as opposed to checking at every time its being set
public static async checkForNewVersionCompatible(hubOrg: Org)